test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -108,16 +107,15 @@ static std::atomic<bool> g_channel_force_delete{false};
 // dns resolution
 
 typedef struct addr_req {
-  grpc_timer timer;
   char* addr;
   grpc_closure* on_done;
   std::unique_ptr<grpc_core::ServerAddressList>* addresses;
 } addr_req;
 
-static void finish_resolve(void* arg, grpc_error_handle error) {
+static void finish_resolve(void* arg) {
```

Comment:
`void* arg` can now just be `addr_req* r`. (or see suggestion below for makign this an `addr_req&& r` argument)

Reply:
Used `(addr_req r)` instead but thanks for the great catch!